src/screens/nano/NanoContractDetail.js

```diff
@@ -144,6 +144,35 @@ function NanoContractDetail(props) {
     });
   }
 
+  const renderBalances = () => {
+    return Object.entries(ncState.balances).map(([token, data]) => {
+      return (
+        <tr key={token}>
+          <td>{data.value}</td>
+          <td>{token === hathorLib.constants.NATIVE_TOKEN_UID ? token : <Link to={`/token_detail/${token}`}>{token}</Link>}</td>
+        </tr>
+      );
+    });
+  }
+
+  const renderNCBalances = () => {
+    return (
+      <div className="table-responsive">
+        <table className="table table-striped table-bordered" id="attributes-table">
+          <thead>
+            <tr>
+              <th className="d-lg-table-cell">Amount</th>
+              <th className="d-lg-table-cell">Token</th>
+            </tr>
+          </thead>
+          <tbody>
+            {renderBalances()}
```

Comment:
suggestion(if-minor): Instead of using a secondary method, just make `Object.entries(...)` here.

Reply:
I usually prefer keeping the map function in a separate method, especially in this file where we have other places that we separate them. Is that ok for you to keep like it is?